powerline-separators.el

```diff
@@ -27,10 +27,25 @@ COLOR1 and COLOR2 must be supplied as hex strings with a leading #."
          (blue (/ (+ (nth 2 c1) (nth 2 c2)) 2)))
     (color-rgb-to-hex red green blue)))
 
+(defun color-xyz-to-apple-rgb (X Y Z)
```

Comment:
sorry it took so long. was about to merge and realized this should be `pl/color-xyz-to-apple-rgb`. So that we have the package prefix there.